threadpoolctl.py

```diff
@@ -774,24 +763,21 @@ def get_threading_layer(self):
         return layer_map[set_threading_layer(-1)]
 
 
-class _OpenMPModule(_Module):
-    """Module class for OpenMP"""
-    def get_version(self):
+class OpenMPController(LibController):
+    """Controller class for OpenMP"""
+    def _get_version(self):
```

Comment:
I think we could keep the get/set methods public on the concrete controller objects.

Reply:
I don't mind